src/api/db/models/Image.js

```diff
@@ -375,6 +376,7 @@ export class ImageModel {
   }
 
   static async createObjects(input, context) {
+    console.log('ImageModel.createObjects - input: ', JSON.stringify(input));
```

Comment:
Creating more consistency in our logging

Reply:
The pattern I am matching for all these methods is input, operations, and res. Let me know if some of these were avoided on purpose.